src/Item_Devices.php

```diff
@@ -1328,7 +1340,9 @@ public static function updateAll($input)
         foreach ($links as $type => $commands) {
             if ($link = getItemForItemtype($type)) {
                 foreach ($commands['add'] as $link_to_add => $number) {
-                    $link->addDevices($number, $itemtype, $items_id, $link_to_add);
+                    if ($link instanceof Item_Devices) {
```

Comment:
Not sure this is needed. It looks like `$link` could only be an Item_Devices at this point.